packages/compass-e2e-tests/tests/logging.test.ts

```diff
@@ -371,6 +371,29 @@ describe('Logging and Telemetry integration', function () {
     });
   });
 
+  describe('on subsequent run', function () {
+    let compass: Compass;
+    let telemetry: Telemetry;
+
+    before(async function () {
+      telemetry = await startTelemetryServer();
+      compass = await beforeTests();
+    });
+
+    after(async function name() {
+      await telemetry.stop();
+      await afterTests(compass);
```

Comment:
Without having dug into the test failures in a ton of detail, I think this *might* fix it and would probably be necessary in order for this test not to be flaky anyway:

```suggestion
      await afterTests(compass);
      await telemetry.stop();
```

Reply:
Thanks!!